inc/functions_modcp.php

```diff
@@ -145,6 +145,22 @@ function send_report($report, $report_type='post')
 {
 	global $db, $lang, $forum, $mybb, $post, $thread, $reputation, $user;
 
+	$report_reason = '';
+	if($report['reasonid'])
+	{
+		$query = $db->simple_select("reportreasons", "title", "rid = '{$report['reasonid']}'", array('limit' => 1));
```

Comment:
Can you cast `$report['reasonid']` to integer here? It comes from an `int` field in the database, but it's cast nonetheless in other places.